README.md

```diff
@@ -21,28 +31,24 @@ proposal, it is not enough to say it would be "nice" or "helpful". Use the
 template to show how Godot is not currently meeting your needs and then
 explain how your proposal will meet a particular need. 
 
-3. Proposals have similar requirements to Pull Requests (PRs) in the main repository.
-The graph in [this article](https://godotengine.org/article/will-your-contribution-be-merged-heres-how-tell)
-outlines what core developers consider when evaluating your PR/proposal.
-
 4. Other users must express interest in your proposal for it to be considered.
```

Comment:
Change the bullet point number to account for the removed bullet point above:

```suggestion
3. Other users must express interest in your proposal for it to be considered.
```

Reply:
It still shows as `4.` for me.